spec.bs

```diff
@@ -395,14 +395,12 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
 
  1. Let |tokens| be the result of [=generate masked tokens|generating masked tokens=] with |numTokens|.
  1. Configure the HTTP request.
-               a. Set a load flag to bypass the HTTP cache.
-          b. Add <a http-header>Sec-Private-State-Token</a> request header containing a
-                      base64-encoded version of the bytestring of tokens
+               1. Set a load flag to bypass the HTTP cache.
+               1. Add <a http-header>Sec-Private-State-Token</a> [=header list/set a structured field value|structured request header=]
+                      containing a base64-encoded version of the bytestring of tokens
```

Comment:
This is an improvement, but there are a few things left to do to make this fully semantic here:

1. You need to add *to* `|request|`'s header list here
2. Pass in a tuple of ([header name](https://fetch.spec.whatwg.org/#header-name) name, [structured field value](https://httpwg.org/specs/rfc8941.html#rfc.section.2) structuredValue).

A good example is in https://wicg.github.io/nav-speculation/prefetch.html#create-navigation-params-by-fetching on step 17.3, I think.

https://w3c.github.io/webappsec-fetch-metadata/#abstract-opdef-set-dest is another example for what this could look like.

It's also worth noting that we should define "base64-encoded version of the bytestring of tokens" more normatively, but that can probably happen in a follow-up